src/modules/dashboard/components/SmiteDialog/SmiteDialogForm.tsx

```diff
@@ -158,23 +297,54 @@ const SmiteDialogForm = ({
             }}
             formattingOptions={{
               numeral: true,
-              prefix: '%',
               // @ts-ignore
               tailPrefix: true,
             }}
             elementOnly
             maxButtonParams={{
               fieldName: 'amount',
-              maxAmount: '10',
+              maxAmount: userPercentageReputation || 0,
```

Comment:
Can't this be:
```suggestion
              maxAmount: String(userPercentageReputation || 0),
```
to satisfy the `string` type requirement in `MaxButtonParams` ?